src/Button/__stories__/ButtonSegmented.stories.js

```diff
@@ -9,9 +9,12 @@ export default {
 };
 
 export const primary = () => (<ButtonSegmented>
-    <Button glyph='survey' />
-    <Button glyph='pie-chart' selected />
-    <Button glyph='pool' />
+    <Button aria-label='Survey' glyph='survey' />
+    <Button
+        aria-label='Pie chart'
+        glyph='pie-chart'
+        selected />
```

Comment:
`selected` was removed

Reply:
It was removed as an attribute of the underlying `<button>` but not as a prop. It still is used to apply a classname.